an-apluss/job_listing_API
# Implement the functionality to create a user

#### What does this PR do?
- Implement `POST  /api/auth/signup` endpoint functionality

#### Description of Task to be completed?
- add dummy data to the database
- add the route pattern to register a new user
- modify the user model
- implement the logic to register a new user

#### How should this be manually tested?
- Clone the repository using `git clone https://github.com/an-apluss/job_listing_API.git` . in your terminal
- run `php -S localhost:8000 -t public` on your terminal to start the server
- make a `POST`  request to the endpoint `localhost:8000/api/auth/signup` with the body request on postman 
- You should see response template like below
````
{
    "data": {
       "first_name": "Marvel",
        "last_name": "Akinseye",
        "user_name": "marvy",
        "is_admin": false,
        "id": 11
       },
     "success": true
 }
````
